pom.xml

```diff
@@ -65,11 +66,6 @@
             <artifactId>vaadin-spring-boot-starter</artifactId>
             <version>${vaadin.version}</version>
         </dependency>
-        
```

Comment:
To remove this, you need an updated vaadin-spring-boot-starter. The rc1 don't have vaadin-spring dependency.

Reply:
We will probably make an RC2 tomorrow with all the changes in flow & maven-plugin & vaadin-spring

Reply:
master branches could point to snapshot versions, so we could test theme before new release?